demos/http/http_demo_s3_download/http_demo_s3_download.c

```diff
@@ -115,6 +149,83 @@
  */
 #define DELAY_BETWEEN_DEMO_RETRY_ITERATIONS_S    ( 5 )
 
+/**
+ * @brief Length in bytes of hash digest.
+ */
+#define SHA256_HASH_DIGEST_LENGTH                ( ( ( uint16_t ) 64 ) )
+
+/**
+ * @brief Buffer Length for AWS IOT Credentials.
+ */
+#define CREDENTIAL_BUFFER_LENGTH                 2048U
+
+/**
+ * @brief AWS Service name to send HTTP request using SigV4 library.
+ */
+#define AWS_SERVICE_NAME                         "s3"
+
+/**
+ * @brief Full AWS S3 server endpoint.
```

Comment:
As this contains the URI oath, it is more than an endpoint
```suggestion
 * @brief Full AWS S3 server URL.
```